packages/extension-ui/src/components/Address.tsx

```diff
@@ -220,15 +220,12 @@ function Address({
                     <span>{parentNameSuri}</span>
                   </div>
                 </div>
-                <div className='name displaced'>
+                <div className='displaced'>
                   <Name />
```

Comment:
`<Name>` also has the class `name` set... It's almost definitely a mistake, because that class contains styles that make no sense to be repeated between a parent and a child.